css/style.css

```diff
@@ -16,4 +17,182 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+ /* Header styles */
+ .header {
+     display: flex;
+     position: fixed;
+     background-color: white;
+     padding-bottom: 5px;
+     width: 100%;
+     border-bottom: 2px solid rgb(234, 235, 236);
+     ;
+ }
 
+ .header-img {
```

Comment:
These values seem to move the logo very far to the right - is this what we're looking for?

![Screenshot 2020-02-07 at 09 11 52](https://user-images.githubusercontent.com/31692/74016172-e6380680-4989-11ea-83c1-4284f1de0aa3.png)

Reply:
I opened the Karma clone website in one window and my copy in another window directly underneath. I attempted to line up the logo as best I could. I'm not sure if this looks different in your browser, @jamesgorrie?